### DISCUSSION TITLE
PHP Fatal error:  Uncaught Error: Class 'Spipu\\Html2Pdf\\Html2Pdf' not found

### DISCUSSION THREAD

**Discussion by mvrand** [composer/composer#9710] [Q&A / Support]
Good evening,
I read a lot on google and tried the suggestions but could not find the right solution.
Do you have an idea for me 
Thank you 

-----
script
    require \_\_DIR\_\_ . '/vendor/autoload.php';
    use Spipu\Html2Pdf\Html2Pdf;
...
    $facture = new \Spipu\Html2Pdf\Html2Pdf('P', 'A4', 'fr');
    $facture->writeHTML($html);
    $facture->output();
-----
result
PHP Fatal error:  Uncaught Error: Class 'Spipu\\Html2Pdf\\Html2Pdf' not found in /var/www/mrm/htdocs/facture_mrm.php:148
-----
intall ok
php composer.phar require spipu/html2pdf
Using version ^4.6 for spipu/html2pdf
./composer.json has been updated
Running composer update spipu/html2pdf
Loading composer repositories with package information
Updating dependencies
Nothing to modify in lock file
Installing dependencies from lock file (including require-dev)
Nothing to install, update or remove
Generating autoload files

**Comment by pgrimaud** [comment]
Hello,

You could try this : 

```php
require __DIR__ . '/vendor/autoload.php';

$facture = new Html2Pdf('P', 'A4', 'fr');
$facture->writeHTML('<h1>Vive la France</h1>');
$facture->output();
```

**Reply by mvrand** [reply]
Thanks for your help
I redirected to the Html2Pdf section

**Comment by stof** [comment]
Issues about using a specific package should be reported in the issue tracker of that package (on https://github.com/spipu/html2pdf in this case), not in the composer repository. Otherwise, that would turn the composer maintainers (only a few people) into a support team for the whole PHP ecosystem, which is not an option.
